example-app/src/main/java/com/hcaptcha/example/MainActivity.java

```diff
@@ -64,10 +71,14 @@ private HCaptchaSize getSizeFromRadio(@NonNull RadioGroup radioGroup) {
         }
     }
 
-    private void initHCaptcha(@Nullable HCaptchaSize size) {
+    private void initHCaptcha() {
+        HCaptchaSize size = getSizeFromRadio(sizeRadioGroup);
+        boolean showDialog = interactive.isChecked();
```

Comment:
Can the `interactive` checkbox be renamed to `Show dialog` instead of `Interactive`?

Reply:
Done